Azure/sonic-buildimage
# [Mellanox][201911] Update FW version to 2008_3110

Signed-off-by: Dror Prital <drorp@nvidia.com>



#### Why I did it
Update FW version to 2008_3110 fixing SN3800 specific warm boot scenario:
- Disable interface
- Warm Boot
- Enable Interface  --> link will remain down. 

#### How I did it
Use new FW that contains the fix  for the problem mentioned above 

#### How to verify it
Run the scenario mentioned above and make sure that the link is up after warm boot 

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


Move to new FW that fix the issue

#### A picture of a cute animal (not mandatory but encouraged)
